run/github-runner/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# syntax=docker/dockerfile:1
+# Best practices: https://docs.docker.com/build/building/best-practices/
+
+FROM ghcr.io/actions/actions-runner:2.322.0
+
+# Add scripts with right permissions.
+USER root
```

Comment:
Seems overkill to run as root just to set a file as executable, I don't think you need to. Can you ask around how to solve that without root?